src/vic3_world/world/vic3_world.h

```diff
@@ -102,6 +102,34 @@ class World
    int playthrough_id_;  // Seed, for deterministic results across conversions for the same series of saves
 };
 
+class WorldBuilder
```

Comment:
I try to aim for one class defined per file. Please move this to its own .h file.